src/lib/schemas/query-search.ts

```diff
@@ -15,7 +16,10 @@ export const querySearchRequestSchema = z.object({
 export const querySearchResponseSchema = z.object({
   query: z.string(),
   formattedResult: z.string(),
+  searchResults: z.any(), // This is a placeholder for the SearchResults type that can't be directly represented in Zod
```

Comment:
That's not true. We **can** represent SearchResults in Zod. Zoom out and look at the greater picture. Make sure we have great correct types and infer them well across the codebase. Clean up all the query search, search results etc. schemas and types.